packages/bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import BpkAnimateHeight from 'bpk-animate-height';
+import { durationBase } from 'bpk-tokens/tokens/base.es6';
+import { cssModules } from 'bpk-react-utils';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import STYLES from './bpk-banner-alert.scss';
+
+const getClassName = cssModules(STYLES);
+
+const animateHeightDuration = parseInt(durationBase, 10);
```

Comment:
Could just be `const duration = parseInt(durationBase, 10);` and reused in both places?

Reply:
Actually, shouldn't we be getting the animate height duration from the animate height component? Ideally through a token? At present it's best guess and could change in either instance.

Reply:
Yes we could.
Our value for `animateHeightDuration` seen here is passed to the animate-height component, so unless we change the value here it won't change!